packages/ontario-frontend-cli/create/questions.js

```diff
@@ -48,6 +48,13 @@ const createQuestions = [
       'What is the file name of the French-language page? (this will also be used for the path: ex. ontario.ca/ma-page-francaise)\n',
     validate: (value) => validFileName(value),
   },
+  {
+    type: 'confirm',
+    name: 'ESLint',
+    message: 'Do you want to use ESLint?\nType "y" or "yes" if you want ESLint\n',
+    default: false,
+    validate: (value) => validYes(value)
+  }
```

Comment:
Take a look at the 'confirm' inquiry type from [inquirer.js](https://www.npmjs.com/package/inquirer#confirm---type-confirm).

Try simplifying the question to something like:
`Add ESLint for finding and fixing code problems? (y/N)`

Reply:
This could also be done using a: [list question](https://www.npmjs.com/package/inquirer#list---type-list)
![inquirerex](https://github.com/ongov/ontario-frontend/assets/112188737/632f618d-aefb-49c6-94be-13a580813ffd)

Reply:
I would still like this to be addressed so that it is not on multiple lines and fits the style we have already set.